docs/migrating-from-factorybot.md

```diff
@@ -0,0 +1,175 @@
+# [WIP] Migrating from FactoryBot
+
+Thoughts/ideas on how to introduce Oaken into your codebase, either alongside or to ultimately replace FactoryBot.
+
+## Outline
+
+1. Get a handle on your object graph
+   - Identify root-level models
+     - Oaken is designed to mirror how your app works
```

Comment:
Really cool that these things resonate! I like to say that a root-level model is typically like Account, Team, Organization.